### PR TITLE
add bootstrap

### PR DESCRIPTION
**Pull request by skoshkarli**

### CODE REVIEW

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: '1'
+    };
+  }
+
+    toggle(tab) {
```
Comment (jbgomez) [inline client/src/components/Header.jsx]: Indentation needs fixing

File: client/src/components/Header.jsx [hunk 2]
```diff
@@ -1,18 +1,69 @@
...
+    return (
+      <header>
+        {(false) ? //this is temporarily set to false. Will need to be modified to check for signed in state
+          (<Navbar className="navbar bg-transparent" light>
+            <NavbarBrand href="/home">
+              <img src={logo} alt="Logo"/>
+            </NavbarBrand>
+            <Nav>
+              <NavItem>
+                <NavLink tag={Link} to='/home'> Home </NavLink>
```
Comment (jbgomez) [inline client/src/components/Header.jsx]: We should remove spaces around ' Home ', unless that was intentional

File: client/src/components/SignUp.jsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import { withRouter } from 'react-router-dom';
 import $ from 'jquery';
+import {Row, Col, Button, Form, FormGroup, Label, Input } from 'reactstrap';
```
Comment (jbgomez) [inline client/src/components/SignUp.jsx]: Not a big deal, but we should try to be consistent with spacing. In this case, the space before and after the {} symbols

File: client/src/components/Header.jsx [hunk 4]
```diff
@@ -15,6 +16,7 @@ class Header extends React.Component {
     this.state = {
       activeTab: '1'
     };
+    console.log("LOCATION", this.props.location)
```
Comment (jbgomez) [inline client/src/components/Header.jsx]: Please remove console log or @cchen146  will have your head on a spike.

Comment (skoshkarli) [reply client/src/components/Header.jsx]: 😂

File: client/src/components/Header.jsx [hunk 5]
```diff
@@ -1,18 +1,69 @@
...
-  )
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: '1'
+  }
+
+    toggle(tab) {
```
Comment (jbgomez) [inline client/src/components/Header.jsx]: Sorry, this line just needs to be unindented once

**Pull request merged**